Quaver.Shared/Config/ConfigManager.cs

```diff
@@ -462,6 +462,11 @@ public static class ConfigManager
         /// </summary>
         internal static Bindable<Keys> KeyEditorIncreaseAudioRate { get; private set; }
 
+        /// <summary>
+        ///     Whether hit accuracies such as Marv/Perf should be displayed
+        /// </summary>
+        internal static Bindable<bool> ShowHitAccuracy { get; private set; }
```

Comment:
The naming for this is very ambiguous. Not sure what it should be at the moment though.